spec/system/admin_create_job_category_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+describe 'Usuário cria categoria de trabalho' do
+  it 'e deve estar logado' do
+    visit job_categories_path
+
+    expect(current_path).to eq(new_user_session_path)
+    expect(page).to have_content('Para continuar, faça login ou registre-se.')
+  end
+
+  it 'e deve ser administrador' do
+    user = create(:user)
+
+    login_as user
+    visit job_categories_path
+
+    expect(page).to have_current_path(root_path)
+    expect(page).to have_content('Você não têm permissão para realizar essa ação.')
+  end
+
+  it 'a partir do menu' do
+    user = create(:user, :admin)
+
+    login_as user
+    visit root_path
+    click_on 'Categorias de trabalho'
+
+    expect(page).to have_current_path(job_categories_path)
+    expect(page).to have_content('Cadastro de categorias de trabalho')
+    expect(page).to have_field('Nome da categoria')
+    expect(page).to have_button('Criar')
+  end
+
+  it 'com sucesso' do
```

Comment:
Normalmente o teste de sucesso mais completo colocamos no início do arquivo, assim eu já tenho a referência do caminho "correto" assim que abro o documento para revisar. Assim eu posso assumir que os próximos são todos variações dessa sequência de passos.